common/tools/dev-tool/src/commands/customization/init.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import path from "path";
+
+import { resolveProject } from "../../util/resolveProject";
+import { createPrinter } from "../../util/printer";
+import { leafCommand } from "../../framework/command";
+import { makeCommandInfo } from "../../framework/command";
+
+import { ensureDir, copy, readdir, stat as fsStat, rm, unlink } from "fs-extra";
+import { prompt } from "../../util/prompt";
+
+const log = createPrinter("initialize-customization");
+
+export const commandInfo = makeCommandInfo(
+  "init",
+  "prepares the project for the customization tool",
+  {
+    generatedDirectory: {
+      shortName: "i",
+      kind: "string",
+      default: "generated",
+      description: "directory containing the generated files. Relative to project root",
+    },
+    srcDirectory: {
+      shortName: "src",
+      kind: "string",
+      default: "src/",
+      description: "directory containing the customized files. Relative to project root.",
+    },
+  },
+);
+
+export default leafCommand(commandInfo, async (options) => {
+  const info = await resolveProject(process.cwd());
+  const srcDirectory = path.join(info.path, options.srcDirectory);
+  const generatedDirectory = path.join(info.path, options.generatedDirectory);
+  log(`Initializing project for customizations`);
+
+  await ensureDir(srcDirectory);
+
+  const srcContents = await readdir(srcDirectory);
+
+  if (srcContents.length > 0) {
+    const answer = await prompt(
```

Comment:
suggestion: can we make the default something we pass to `prompt` so we can have it just return that directly instead of having to inspect the result?